### ISSUE TITLE
Question: QR code

### ISSUE THREAD

**Issue by abhatikar** [edgeimpulse/mobile-client#2]
Hi,

I tried to run this client locally but I cant see the QR code getting generated. Am i missing something ?

thanks

<img width="1359" alt="Screenshot 2020-06-08 at 11 45 11" src="https://user-images.githubusercontent.com/7994511/84022097-a6a41a00-a97d-11ea-8ae8-49d84fb09332.png">

**Comment by abhatikar** [comment]
Tried on Mac and Ubuntu. Same issue.

**Comment by janjongboom** [comment]
@abhatikar Yeah this is referring to the QR code you can scan from the **Devices** page in Edge Impulse. If you want to run the client locally you need to append your API key to the URL:

https://whereveryouruntheservice/?apiKey=ei_XXXX

And the message will go away.

**Comment by abhatikar** [comment]
I did not get the qr code, adding the parameter connects to the platform.
Thanks
